core/trino-main/src/main/java/io/trino/exchange/ExchangeManagerRegistry.java

```diff
@@ -69,7 +69,7 @@ public void loadExchangeManager()
 
     public synchronized void loadExchangeManager(String name, Map<String, String> properties)
     {
-        log.info("-- Loading exchange manager %s --", name);
+        log.debug("-- Loading exchange manager %s --", name);
```

Comment:
this is not test only code and is very useful in production to verify what exchange managers were actually loaded.

cc: @losipiuk

Reply:
I guess this one can be `debug` 
But please revert `log.debug("-- Loaded exchange manager %s --", name);` to `info`.

Reply:
See https://github.com/trinodb/trino/pull/19554#issuecomment-1790828066